pennylane_lightning/core/src/simulators/lightning_gpu/algorithms/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ target_link_libraries(${PL_BACKEND}_algorithms PRIVATE      lightning_compile_op
                                                             )
 
 target_link_libraries(${PL_BACKEND}_algorithms PUBLIC       ${PL_BACKEND}_utils
+                                                            cuda_utils
```

Comment:
couldn't this be included in `{PL_BACKEND}_utils` for only `PL_BACKEND`s with cuda support?

Reply:
`{PL_BACKEND}_utils` should include backend specific headers, and hence no `cuda_utils`. We could include it in `lightning_utils` for backends with CUDA support however.

Reply:
Removed.